test/unit/presenters/publishing_api/ministers_index_presenter_test.rb

```diff
@@ -0,0 +1,102 @@
+require "test_helper"
+
+class PublishingApi::MinistersIndexPresenterTest < ActionView::TestCase
+  def present(options = {})
+    PublishingApi::MinistersIndexPresenter.new
+  end  
+  
+  test "presenter is valid against ministers index schema" do
+    create(:sitewide_setting, key: :minister_reshuffle_mode, on: true)
+
+    presented_item = present()
+
+    print presented_item.content
+    assert_valid_against_schema(presented_item.content, "ministers_index")
+  end
```

Comment:
Since the `present` method returns an instance instead of performing an action, it'd be better to call it with a noun rather than a verb. That's more accurate and clearer to the reader.
You could call it `presented_item` and avoid creating the variable.